homeassistant/components/tautulli/const.py

```diff
@@ -1,11 +1,14 @@
 """Constants for the Tautulli integration."""
 from logging import Logger, getLogger
+from typing import Final
+
+ATTR_TOP_USER = "top_user"
 
 CONF_MONITORED_USERS = "monitored_users"
 DEFAULT_NAME = "Tautulli"
 DEFAULT_PATH = ""
 DEFAULT_PORT = "8181"
 DEFAULT_SSL = False
 DEFAULT_VERIFY_SSL = True
-DOMAIN = "tautulli"
+DOMAIN: Final = "tautulli"
```

Comment:
If does not make sense to use that for just one of these.
Since its not related to the goal of this PR, revert this change